.github/workflows/assemble.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17, 21 ]
```

Comment:
This change looks good to me. When should we take the java versions < 21 out of this matrix on main?

Reply:
Thanks @cwperks , see please https://github.com/opensearch-project/OpenSearch/pull/16366